client/components/jetpack/card/jetpack-product-card/display-price/paid.tsx

```diff
@@ -47,14 +56,42 @@ const Paid: React.FC< OwnProps > = ( {
 	}
 
 	const renderDiscountedPrice = () => {
+		/*
+		 * Price should be displayed from left-to-right, even in right-to-left
+		 * languages. `PlanPrice` seems to keep the ltr direction no matter
+		 * what when seen in the dev docs page, but somehow it doesn't in
+		 * the pricing page.
+		 */
+		if ( isPricingPageTreatment202204 ) {
+			return (
+				<>
+					<span dir="ltr">
+						<PlanPrice
+							discounted
+							className="display-price__discounted-price"
+							rawPrice={ finalPrice as number }
+							currencyCode={ currencyCode }
+							displayShortPerMonthNotation
+						/>
+					</span>
+					<br />
+					<span dir="ltr">
+						<PlanPrice
+							original
+							className="display-price__original-price display-price__original-price-small"
+							rawPrice={ originalPrice as number }
+							currencyCode={ currencyCode }
+							originalPricePrefix={ translate( 'normally', {
+								comment: 'A way to describe a price before a discount is applied',
+							} ) }
+						/>
+					</span>
+				</>
+			);
+		}
+
 		return (
 			<>
-				{ /*
```

Comment:
I think this comment should be left in place.

Reply:
The comment still exists, it's now just above the if statement that conditionally shows the test variation, since it applies to both cases.

Reply:
Woops, I misread that 🙃